GPU/GPUTracking/TPCClusterFinder/DecodeZS.h

```diff
@@ -15,18 +15,54 @@
 #define O2_GPU_DECODE_ZS_H
 
 #include "clusterFinderDefs.h"
-#include "GPUTPCClusterFinderKernels.h"
+#include "GPUGeneralKernels.h"
+#include "GPUConstantMem.h"
+#include "DataFormatsTPC/ZeroSuppression.h"
 
 namespace GPUCA_NAMESPACE
 {
 namespace gpu
 {
 
 class GPUTPCClusterFinder;
-class DecodeZS
+
+class DecodeZS : public GPUKernelTemplate
 {
  public:
-  static GPUd() void decode(GPUTPCClusterFinder& clusterer, GPUTPCClusterFinderKernels::GPUTPCSharedMemory& s, int nBlocks, int nThreads, int iBlock, int iThread);
+  class GPUTPCSharedMemory
```

Comment:
Could you please make this a struct here to avoid the naming convetion problems in the codechecker.

Reply:
Fixed it.